packages/contracts-bedrock/contracts/universal/Semver.sol

```diff
@@ -9,17 +11,17 @@ contract Semver {
     /**
      * @notice Contract version number (major).
      */
-    uint256 public immutable MAJOR_VERSION;
+    uint256 private immutable MAJOR_VERSION;
 
     /**
      * @notice Contract version number (minor).
      */
-    uint256 public immutable MINOR_VERSION;
+    uint256 private immutable MINOR_VERSION;
```

Comment:
Will making these uint8s make the bytecode smaller?

Reply:
Probably, but not significantly.